src/runtime/mod.rs

```diff
@@ -13,7 +13,16 @@ thread_local! {
     pub(crate) static CONTEXT: RuntimeContext = RuntimeContext::new();
 }
 
-/// The Runtime executor
+/// The Runtime Executor
+///
+/// This is the Runtime for `tokio-uring`.
+/// It wraps the default [`Runtime`] using the platform-specific Driver.
+///
+/// # Multithreading
+///
+/// This executes futures and tasks within the current-thread only.
```

Comment:
So, I don't see a need to mirror what is in the tokio doc. Why not remove the `# Multithreading` and just make sure somewhere in this file's doc is says the current state is `current-thread` tokio scheduling only. Of course if you want to get this in quickly now, you may be better off just nixing this change to the comments for now.

Reply:
K.